src/spec-node/devContainersSpecCLI.ts

```diff
@@ -507,6 +507,22 @@ function buildHandler(args: BuildArgs) {
 	(async () => build(args))().catch(console.error);
 }
 
+function removeDuplicateImages(images: string[]): string[] {
```

Comment:
Why did we add this?

The source code was correctly fixing the bug, the only issue was the way how the tests were constructed.
Can we remove the latest source code changes and only update the tests to correctly validate the bug fix changes?